templates/zerver/help/upload_share_files.md

```diff
@@ -0,0 +1,26 @@
+## Upload and Share Files
+
+The paperclip icon (![paperclip](/static/images/help/paperclip.png)) under the message field lets you attach files, including images, sound, and video. Alternatively, you may also drag and drop files on the message field. The files are uploaded to a server and a link is supplied, but a thumbnail is displayed if possible. You’ll see the link in Markdown format: `[link_text](link_URL)`
```

Comment:
May want to break this paragraph up visually to have each option for action appear under a bullet. (This text may need some work):
e.g.

Zulip allows you to easily attach files, including images, sound, and video by:
- *Clicking the Paperclip Icon* Use the he paperclip icon (\!\[paperclip\]\(/static/images/help/paperclip.png\))
- *Dragging* You may also drag and drop files on the message field

The files are uploaded to a server and a link is supplied, but a thumbnail is displayed if possible. The link will be in Markdown format: `[link_text](link_URL)`